recipes/sam3/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "sam3" %}
+{% set git_hash = "84cc43bca4347b772f17d1078a1ddb4c054655c2" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/facebookresearch/sam3/archive/{{ git_hash }}.tar.gz
+  sha256: b50d5f85bc57f13ef8c1b7fe485688bc0f76f58d18078ce9b505acb3bcfa408b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=64
+    - wheel
+    - pip
+  run:
+    - python >={{ python_min }}
+    - timm >=1.0.17
+    - numpy >=1.26,<2.4.0
+    - tqdm >=4.66.0,<4.68.0
+    - ftfy >=6.1.1,<6.4.0
+    - regex >=2025.11.3
+    - iopath >=0.1.10,<0.2.0
+    - typing_extensions >=4.8.0,<4.16.0
+    - huggingface_hub >=0.30.0,<1.0
+    - einops >=0.8.0,<0.9.0
```

Comment:
I'm curious where you got all these precise bounds on things?

i'm more interested in avoiding these kinds of "arbitrary bounds".

They add maintenance, and if they aren't required, then just loosen the

Reply:
I got those bounds from the unofficial sam3 package on PyPI
 https://pypi.org/project/sam3/#files

<img width="399" height="278" alt="image" src="https://github.com/user-attachments/assets/a5bcd3c0-0b79-44dc-93fc-a1544b297bde" />

I have loosened the bounds to be the same as the official sam3 by Meta
https://github.com/facebookresearch/sam3/blob/main/pyproject.toml#L27
<img width="278" height="253" alt="image" src="https://github.com/user-attachments/assets/ee5a4bc9-e072-4cd6-b837-b16cde579c0f" />